compose.yml

```diff
@@ -184,32 +188,28 @@ services:
 
     updateEFO:
         image: python:3.7
-        env_file:
-            - ./.env
-            - ./.env.secret.updateEFO
         restart: unless-stopped
         profiles:
             - production
         volumes:
             - ./updateEFO:/updateEFO
             - ./_logs/updateEFO:/var/log/updateEFO
             - ./updateEFO/rotatelogs-compress.sh:/rotatelogs-compress.sh
+        secrets:
+            - efo_secrets
         entrypoint:
             - /updateEFO/updateEFO.sh
 
     restic:
         image: ghcr.io/wikiteq/restic-backup-docker:stream
-        env_file:
-            - ./.env
-            - ./.env.secret.restic
         profiles:
             - production
         restart: unless-stopped
         links:
             - db
         environment:
             - MW_DB_USER=root
-            - MW_DB_PASS=$MW_DB_INSTALLDB_PASS
+            - MW_DB_PASS_FILE=/run/secrets/db_root_password
```

Comment:
Won't work by the same reason - there is nothing that uses `MW_DB_PASS_FILE` env to read a value from a file and store into `MW_DB_PASS`

Reply:
It will work

Reply:
Same as https://github.com/WikiWorks/BugSigDB/pull/51#discussion_r1883931412